### DISCUSSION TITLE
BUG: After the last update 3.0.36

### DISCUSSION THREAD

**Discussion by sbc640964** [filamentphp/filament#8129] [General]
After the last update 3.0.36
Error opening modals
```
Uncaught TypeError: Cannot read properties of null (reading 'nextSibling') at get children [as children]
```

**Comment by danharrin** [comment]
Cannot reproduce.

**Reply by sbc640964** [reply]
@danharrin

Amazing!
I discovered a TextColumn with formatStateUsing that return HTML that was missing a tag bracket.
Interestingly, until version 3.0.31 it worked...
